eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -63,17 +76,25 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
         parameters:
           NuGetCacheKey: Test
+      - pwsh: | 
+          if ("${{ parameters.ProjectListOverrideFilePropertyName }}" -ne '') {
+            Write-Host "##vso[task.setvariable variable=ProjectListFileName]$(${{ parameters.ProjectListOverrideFilePropertyName }})"
```

Comment:
Do you want to handle whether the variable itself is empty or not and log an error if so? Also if the variable could be empty, it's better to use single quotes, otherwise devops does not replace the `$(foo)` expression and powershell hits an error of its own trying to evaluate it.

Reply:
I think we can avoid this step completely if we just access the property as mentioned below.